src/store_client.cc

```diff
@@ -193,12 +195,29 @@ storeClientCopyEvent(void *data)
     storeClientCopy2(sc->entry, sc);
 }
 
+void
+store_client::initReplyBuffer()
+{
+    if (!replyBuffer) {
+        replyBuffer = new MemBuf;
+        replyBuffer->init();
```

Comment:
This uses default limits. Can we be more precise here? Perhaps using Config.maxReplyHeaderSize?

BTW, please check whether the same concern applies to parsing responses in the shared memory cache (#72). I forgot to do that.